templates/events/details.html

```diff
@@ -189,7 +189,24 @@ <h2 id="event-details-heading">
                                     <p class="status-text">{{ user_status.message }}</p>
                                 {% endif %}
                             {% else %}
-                                <p class="status-text">{{ user_status.message }}</p>
+                                <p class="status-text">{{ user_status.message }}
+									<br><button type="button" class="btn btn-primary" data-toggle="modal" data-target="#myModal">Info</button></p>
+									<div id="myModal" class="modal fade" role="dialog" style="color:#666;">
```

Comment:
I was in a hurry when I tried to implement the solution and I may have overlooked the naming. The inline-style was a quick-fix for the text color (is there an existing class for text colors?). I will try to make it look a bit more professional than this.